shared/src/main/java/com/example/cosc345/shared/models/Product.kt

```diff
@@ -65,8 +67,8 @@ data class Product(
             }.keys
 
         val nonLocalPrices = information!!.filter {
-            localRetailers.contains(it.retailer) && it.pricing!!.any { (pricingStore) ->
-                retailers[it.retailer]!!.stores!!.first { (id) -> id == pricingStore }.region.equals(
+            localRetailers.contains(it.retailer) && it.pricing!!.any { (storeId, _, _, _, _, _, _, _) ->
+                retailers[it.retailer]!!.stores!!.first { (id, _, _, _, _, _, _) -> id == storeId }.region.equals(
```

Comment:
Is it possible to do the filtering without the underscores?

Reply:
blame android studio linter, it says I should do this for some reason